Code/src/main/java/nl/utwente/viskell/haskell/env/CatalogFunction.java

```diff
@@ -12,16 +12,20 @@
     /** The documentation string for this Entry. */
     private final String documentation;
 
+    /** Whether this function is a constructor **/
+    private boolean isConstructor;
```

Comment:
Could probably be `final`. Also, are constructors the only functions-that-aren't-really-functions we are going to get? If not, we might need some other way to model the different kinds.